src/analytics/Properties.tsx

```diff
@@ -1311,6 +1312,27 @@ interface BuilderHooksProperties {
   [BuilderHooksEvents.hooks_disable_preview]: undefined
 }
 
+interface DappShortcutClaimRewardEvent {
+  rewardId: string
+  appName: string
+}
+interface DappShortcutsProperties {
+  [DappShortcutsEvents.dapp_shortcuts_rewards_screen_open]: {
+    numRewards: number
+  }
+  [DappShortcutsEvents.dapp_shortcuts_reward_claim_start]: DappShortcutClaimRewardEvent & {
+    rewardTokens: string[]
+    rewardAmounts: string[]
```

Comment:
I forgot if arrays are easy to analyze in our analytics pipeline.

Reply:
i'm also not sure, i asked the question in the #data channel. i changed these to csv's to be safe, although i do see us using string[] elsewhere